docs/openapi3.yaml

```diff
@@ -1723,3 +1956,32 @@ components:
             timeout:
               description: timout to smtp server in milliseconds
               type: number
+    download:
+      description: Download processor file from the file_url and store it.
+      required:
+      - file_url
+      properties:
+        file_url:
+          type: string
+          description: url that the file can be downloaded from, should be direct download link.
+        force_download:
```

Comment:
move this parameter to be under `Job` resource. If job has a processor_file, then user has the ability to force download it or not. (Default: false)